src/main/resources/entity-types/inventory/simple_instance.json5

```diff
@@ -1068,6 +1069,8 @@
       source: {
         columnName: 'source_tenant_id',
         entityTypeId: '8fc4a9d2-7ccf-4233-afb8-796911839862',
+        type: 'fqm',
```

Comment:
Note: the other available `type` is `entity-type`. I made that the default, though, which is why this PR doesn't modify every single `source` in the project